nginx-fem-redirects.conf

```diff
@@ -1,4 +1,23 @@
 set $fe_project_uri "https://fe-project.zooniverse.org";
+
+location ~* ^/(?:projects|about)/_next/.+?$ {
+    resolver 1.1.1.1;
+    proxy_pass $fe_project_uri;
+    proxy_set_header Host fe-project.zooniverse.org;
+}
+
+location ~* ^/(?:projects|about)/assets/.+?$ {
+    resolver 1.1.1.1;
+    proxy_pass $fe_project_uri;
+    proxy_set_header Host fe-project.zooniverse.org;
+}
+
+location ~* ^/p/assets/.+?$ {
```

Comment:
Do we have a path that starts with a `/p/` ?

Reply:
I think this is what happens when writing code while feeling the side effects of the vaccine.